README.md

```diff
@@ -206,6 +206,12 @@ Body:
     }
 ]
 ```
+### Deletes a food from a meal in the database
+Request:
+```
+DELETE /api/v1/meals/:meal_id/foods/:id
+Accept: application/json
+```
```

Comment:
Need to add what the successful / failed responses are, even if it's just status codes.

Reply:
Resolved on branch